### DISCUSSION TITLE
Change the scope of a bot to applications.command

### DISCUSSION THREAD

**Discussion by MateoMROZEK** [discordjs/discord.js#7505] [Q&A] [answered]

**Comment by Drxckzyz** [comment]
If they just invite the bot to the server (click the link and invite it without kicking it) with the scope it will refresh the applications with the new permissions and scopes

**Reply by MateoMROZEK** [reply]
I don't really understand, basically I have some servers that don't have my /slash, even though I put slashes. I have a 403 "not permission" error, so I want to try to have a way for the servers that don't have these slash commands to have them without them deleting the bot from that server.

**Reply by Drxckzyz** [reply]
I said permission to just say what inviting the bot without kicking it does, Anways you should read advaith’s answer below as it goes into details

**Comment by advaith1** [comment, marked as answer]
you won't get a 403 on global command creation; looks like you're trying to create a guild command in a specific guild

and the server admins will need to add your application with the applications.commands scope, the application can't give it to itself because then it would be pointless. note that **they do not have to kick the bot**, they can simply authorize it with the applications.commands scope without removing it.

Discord did a scope migration in March 2021 (so if the application was authorized in a server with the bot scope then it automatically got the applications.commands scope) and they might do another one after perms v2 releases.

You should update the add link you're giving to users and add the applications.commands scope if you have not already.

**Comment by almostSouji** [comment]
Not too sure why you delete your question and unmark and thumbs-down answers that solve your problem?
The server you want to add commands to needs to invite the bot again with the proper scopes, without removing it first.
They can just click the invite again, it will update the scope.

If you want that to be the answer or not is pretty irrelevant, because it is the answer to your issue.
